src/Config/Converter.php

```diff
@@ -26,6 +26,8 @@ class Converter implements ConverterInterface
             'delete_old_files' => ['type' => 'bool', 'default' => false],
             'archive_date_format' => ['type' => 'string', 'default' => 'dmYhis'],
             'directory_permissions' => ['type' => 'int', 'default' => 0777],
+            'archive_already_imported_files' => ['type' => 'bool', 'default' => false],
+            'process_only_last_file' => ['type' => 'bool', 'default' => false],
```

Comment:
What is advantage of that? What is a use case for that? If there is only one file added to import dir, than this is not really needed. If there are multiple file we can stuck on last file, until we upload new one - but some files will be skipped. Does that make any sense? I mean similar effect can be achieved by archiving processed file. 

What you think?

Reply:
Use case for this is:
When you have multiple csv files and all of them have full dump of e.g. products, you don't want to process all of them and run multiple updates of the same products. You just need to process only the latest file of that collection.

Our case is that vendor is uploading full product export csv many times a day, but we need to process it only twice a day and only the latest uploaded file.

Reply:
Ok, so if merchant adds incremental updates only the last file will be picked and other that were not processed will be given up? Is the idea of that?

Reply:
for incremental (diff) updates it should be turned off

Reply:
Ok sounds very useful. Could you describe use cases in readme. Are these new fields available for all source types?